test/conformance/tests/test_sriov_operator.go

```diff
@@ -1157,6 +1157,39 @@ var _ = Describe("[sriov] operator", func() {
 						allocatable, _ := resNum.AsInt64()
 						return allocatable
 					}, 10*time.Minute, time.Second).Should(Equal(int64(5)))
+
+					By("Creating sriov network to use the vfio device")
+					sriovNetwork := &sriovv1.SriovNetwork{
+						ObjectMeta: metav1.ObjectMeta{
+							Name:      "test-vfionetwork",
+							Namespace: operatorNamespace,
+						},
+						Spec: sriovv1.SriovNetworkSpec{
+							ResourceName:     resourceName,
+							IPAM:             `{"type":"host-local","subnet":"10.10.10.0/24","rangeStart":"10.10.10.171","rangeEnd":"10.10.10.181","routes":[{"dst":"0.0.0.0/0"}],"gateway":"10.10.10.1"}`,
+							NetworkNamespace: namespaces.Test,
+						}}
+
+					err = clients.Create(context.Background(), sriovNetwork)
+					Expect(err).ToNot(HaveOccurred())
+					Eventually(func() error {
```

Comment:
can you use `waitForNetAttachDef()`?

https://github.com/k8snetworkplumbingwg/sriov-network-operator/blob/ee2ee7422b0ca4ea811e242f1a5d2fcc012a36fd/test/conformance/tests/test_sriov_operator.go#L2819

Reply:
you are right!

done